templates/views/index.pug

```diff
@@ -95,5 +88,3 @@ block content
 block js
   script const refreshCountersSeconds = !{refreshCountersSeconds};
   script(src="/js/app/counter.js")
-  script(src="/js/world-map.min.js")
-  script(src="/js/world-map-dependencies.min.js")
```

Comment:
As far as I know there is a whole lot of CSS that needs to be deleted in addition to the javascript.